Formula/ignition-utils0.rb

```diff
@@ -0,0 +1,53 @@
+class IgnitionUtils0 < Formula
+  desc "General purpose classes and functions designed for robotic applications."
+  homepage "https://github.com/ignitionrobotics/ign-utils"
+  url "https://github.com/ignitionrobotics/ign-utils/archive/20527970a7795661d8ca7b15f8197e4457595ce5.tar.gz"
+  version "0.1.0~pre1"
+  sha256 "fcb84b71063b7a2a9a0b53e31aa57101e1dc4005e2d38487e046bdec2849ec46"
+  license "Apache-2.0"
+
+  depends_on "cmake"
+  depends_on "ignition-cmake2"
+  depends_on "pkg-config"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS
+      #include <ignition/utils/ImplPtr.hh>
+      class SomeClass
+      {
+        private: ignition::utils::ImplPtr<Implementation> dataPtr;
```

Comment:
You'll need an `Implementation` somewhere above for this to work.

Reply:
fd7c0bd0ed4e37a6c95ac2aaf2f26306e5b3a853